apps/prairielearn/src/middlewares/content-security-policy.ts

```diff
@@ -0,0 +1,12 @@
+import { Router } from 'express';
+
+const router = Router();
+
```

Comment:
Nit: I like the long spelled out name for findability, but we also might want to put CSP in a comment or something in case someone is searching the codebase for that?

Reply:
Done somewhat-organically in f598b35.